hack/deploy-karmada-opensearch.sh

```diff
@@ -42,16 +42,15 @@ fi
 TEMP_PATH=$(mktemp -d)
 cp $HOST_CLUSTER_KUBECONFIG $TEMP_PATH/kubeconfig
```

Comment:
cp a temp kubeconfig is unnecessary if we don't change it

Reply:
done